include/fmt/format.h

```diff
@@ -943,17 +969,7 @@ FMT_EXTERN template struct basic_data<void>;
 // This is a struct rather than an alias to avoid shadowing warnings in gcc.
 struct data : basic_data<> {};
 
-#ifdef FMT_BUILTIN_CLZLL
-// Returns the number of decimal digits in n. Leading zeros are not counted
-// except for n == 0 in which case count_digits returns 1.
-inline int count_digits(uint64_t n) {
-  // https://github.com/fmtlib/format-benchmark/blob/master/digits10
-  auto t = bsr2log10(FMT_BUILTIN_CLZLL(n | 1) ^ 63);
-  return t - (n < data::zero_or_powers_of_10_64_new[t]);
-}
-#else
-// Fallback version of count_digits used when __builtin_clz is not available.
-inline int count_digits(uint64_t n) {
+template <typename T> FMT_CONSTEXPR int count_digits_trivial(T n) {
```

Comment:
But it's used both for `uint32_t` and `uint64_t`, not for `uint128_t` yet only because I was stopped by upper-case literal `U` there... looks like it has the same meaning as `u` literal, so even `uint128_t` can use this function. That's why I have template there.

Reply:
What about `template` here, should I expand usage of `count_digits_fallback` to `count_digits` for `uint128_t`?

Reply:
Let's keep it as a template since it's just a fallback.